pages/about.vue

```diff
@@ -85,29 +85,29 @@
     "logoTypeGothic": "ロゴたいぷゴシック"
   },
   "zh-CN": {
-    "aboutTitle": "About this website",
-    "aboutDescription": "About the Genshin Dictionary. This website is an online English-Chinese-Japanese dictionary for the proprietary terms used in a computer game Genshin Impact.",
-    "aboutIntro": "This website is an online English-Chinese-Japanese dictionary for the proprietary terms used in a computer game {genshinLink}.",
-    "wordCount": "Currently, this dictionary has {count} words.",
-    "precautionsTitle": "Notice and Disclaimer",
-    "precautions1": "Since the translations on this website are the proprietary terms for a computer game, the most of the words are not literally translated.",
-    "precautions2": "This website is fanmade and not affiliated or endorsed by miHoYo and HoYoverse (Cognosphere).",
-    "operatorTitle": "Owner and Contributors",
-    "operator": "Owner: Xicri ({twitterLink} / {githubLink})",
-    "contributor": "Initial Chinese translation: Bill Haku ({bilibiliLink} / {twitterLink} / {githubLink})",
-    "contact1": "Contact the owner on {twitterLink}. (DM available)",
-    "contact2": "If you have any bug report, questions for opendata/API, or any other technical inquiries, GitHub {issuesLink} and  {discussionsLink} are also available for the contact.",
-    "contact3": "Please use English or Japanese as possible.",
-    "creditsTitle": "Credits",
-    "credits1": "Japanese pronunciation data for some words are derived from {genshinDictionaryLink} and {genshinKanjiLabLink}.",
-    "credits2": "Some translations for the Artifacts added on the early days of this website is referrenced from {genshinWikiLink}.",
-    "credits3": "This website uses {octiconsLink} provided by GitHub, Inc. under {licenseLink}",
-    "credits4": "The favicon of this website use {logoTypeGothicLink}.",
+    "aboutTitle": "关于本站",
+    "aboutDescription": "关于本站：本站是一个在线的中日英三语词典，收录了《原神》中使用的专有词汇。",
+    "aboutIntro": "本站是一个在线的中日英三语词典，收录了《{genshinLink}》中使用的专有词汇。",
+    "wordCount": "截至目前，本站已收集了 {count} 个专有词汇。",
+    "precautionsTitle": "使用注意事项及免责声明",
+    "precautions1": "由于本站收录的是《原神》中的专有词汇，因此大多数单词并非直译。",
+    "precautions2": "本站是一个由《原神》玩家制作并运营的网站，与开发商上海米哈游网络科技股份有限公司及 Cognosphere Pte., Ltd. (HoYoVerse) 无关。",
```

Comment:
```suggestion
    "precautions2": "本站是一个由《原神》玩家制作并运营的网站，与开发商上海米哈游网络科技股份有限公司及 Cognosphere Pte., Ltd. (HoYoverse) 无关。",
```

Looks like HoYo"v"erse (lowercase v) is official name.

https://www.hoyoverse.com/en-us/about-us